lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -183,8 +194,70 @@ class RenderForm extends React.Component {
     );
   }
 
+  async handleInputChange(event) {
+    event.preventDefault();
+    const queryString = event.target.value;
+    if (this.typingTimeout) clearTimeout(this.typingTimeout);
+    this.typingTimeout = setTimeout(async () => {
+      if (queryString.length > 2) {
+        let suggestions = [];
+        await fetch(`${this.learnerSupportCenterURL}/hc/api/internal/instant_search.json?query=${queryString}&locale=en-us`, { headers: {
+          'Content-Type': 'application/json',
+        } }).then((response) => {
+          suggestions = response.json.results;
+          this.setState({ suggestions });
```

Comment:
Is `this` component or another object? can you verify? Because the code here is nested and JS can act weird with this in nested object structure.

Reply:
I've thoroughly verified the behavior. `this` is always an object of `RenderForm`.
<img width="735" alt="Screenshot 2021-05-28 at 11 05 40 PM" src="https://user-images.githubusercontent.com/52413434/120024857-3b79aa80-c009-11eb-9fd9-c8f39ce2df5d.png">